sunpy/net/fido_factory.py

```diff
@@ -351,7 +351,7 @@ def fetch(self, *query_results, **kwargs):
         --------
         >>> from sunpy.net.vso.attrs import Time, Instrument
         >>> unifresp = Fido.search(Time('2012/3/4','2012/3/5'), Instrument('EIT'))  # doctest: +REMOTE_DATA
-        >>> downresp = Fido.fetch(unifresp)  # doctest: +SKIP
+        >>> downresp = Fido.fetch(unifresp)  # doctest: +REMOTE_DATA
```

Comment:
keep the skip here, we don't want to be downloading more data from the VSO than we need to.